scheduler/replica_scheduler_test.go

```diff
@@ -1689,3 +1689,43 @@ func getTestNow() time.Time {
 	now, _ := time.Parse(time.RFC3339, TestTimeNow)
 	return now
 }
+
+func (s *TestSuite) TestScheduleReplicaToDiskOnLocalNode(c *C) {
+	rs := NewReplicaScheduler(nil)
+	volume := newVolume(TestVolumeName, 2)
+	replica1 := newReplicaForVolume(volume)
+	replica1.Spec.HealthyAt = getTestNow().String()
```

Comment:
IIRC, we don't need to set `Spec.HealthyAt` for this kind of newly created replica.

Reply:
`newReplicaForVolume` function doesn't set `HealthyAt` field, but it is required to pass the following if statement inside the `scheduleReplicaToDiskOnLocalNode` function:
```golang
		if r.Spec.NodeID == localNodeID && r.Spec.HealthyAt != "" && r.Spec.FailedAt == "" {
			return
		}
```
The purpose of this if statement is to find a healthy replica that is already assigned to the local node. Maybe, we could change this if statement to look like this?
```golang
		if r.Spec.NodeID == localNodeID && r.Spec.FailedAt == "" {
			return
		}
```

Reply:
Typically, `r.Spec.HealthyAt` will be set when the replica is started as a healthy one of an attached volume for the first time.

> The purpose of this if statement is to find a healthy replica that is already assigned to the local node. Maybe, we could change this if statement to look like this?

Yes. This change should make more sense.